### PR TITLE
add machakos wards

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: core_data_modules/cleaners/codes/kenya_codes.py [hunk 1]
```diff
@@ -342,7 +342,92 @@ class KenyaCodes(object):
     WUNDANYI = "wundanyi"
     YATTA = "yatta"
 
-    CONSTITUENCY_TO_COUNTY_MAP = {
+    # Kenya Wards
```
Comment (as2388) [inline core_data_modules/cleaners/codes/kenya_codes.py:345]: Could you add a note saying that this list is incomplete?

Comment (as2388) [inline core_data_modules/cleaners/codes/kenya_codes.py:345]: Can you add a link to where the data came from too if there's one available?

**Review by as2388 — COMMENTED** [review]
Thanks, I'll review again when the tests are fixed

**Review by DanielMwendwa — APPROVED** [review]

File: core_data_modules/cleaners/codes/kenya_codes.py [hunk 2]
```diff
@@ -342,298 +342,388 @@ class KenyaCodes(object):
...
+    MATUNGULU_WEST = "matungulu west"
+    MITABONI = "mitaboni"
+    MUA = "mua"
+    MUTITUNI = "mutituni"
+    MACHAKOS_CENTRAL = "machakos central"
+    MUMBUNI_NORTH = "mumbuni north"
+    MUVUTINI_KIIMA_KIMWE = "muvutini/kiima-kimwe"
+    MBIUNI = "mbiuni"
+    MAKUTANO = "makutano"
+    MASII = "makutano"
```
Comment (DanielMwendwa) [inline core_data_modules/cleaners/codes/kenya_codes.py:376]:
```suggestion
    MAKUTANO = "makutano"
    MASII = "masii"
```

File: core_data_modules/cleaners/codes/kenya_codes.py [hunk 3]
```diff
@@ -342,298 +342,388 @@ class KenyaCodes(object):
...
+    NDALANI = "ndalani"
+    SYOKIMAU_MULOLONGO = "syokimau/mulolongo"
+    TALA = "tala"
+    UPPER_KAEWA_IVETI = "upper kaewa/iveti"
+    WAMUNYU = "wamuyu"
+
+    # Machakos wards -> consituency map
+    # Todo: update the rest of the wards
+    WARD_TO_CONSTITUENCY_MAP = {
+            KIVAA: MASINGA,
```
Comment (as2388) [inline core_data_modules/cleaners/codes/kenya_codes.py:391]: Indentation, here and in the next dict
```suggestion
    WARD_TO_CONSTITUENCY_MAP = {
        KIVAA: MASINGA,
```

**Review by as2388 — APPROVED** [review]

**Pull request merged**